### ISSUE TITLE
[JavaScript] prints may crash on internet explorer

### ISSUE THREAD

**Issue by fungiboletus** [TelluIoT/ThingML#296]
`console.log.call` does not work very well on internet explorer 11.

**Comment by fungiboletus** [comment]
Just to make things more fun, the behaviour is different whether the developer tools are open or not.

**Comment by brice-morin** [comment]
I am tempted to tag it as `won't fix` :-)

**Comment by fungiboletus** [comment]
Yeah, I miss SINTEF for that. Supporting shitty browsers for money isn't fun.
